### PR TITLE
Add Breadcrumbs

### PR DESCRIPTION
**Pull request by FlHinrichsen**

### CODE REVIEW

**Comment by bruno-garcia** [comment]
@FlHinrichsen Thanks for contributing!
Before we can merge this it'd be good to have some tests and exemplify the usage of the feature on the sample project.

File: src/Sentry.Log4Net/SentryAppender.cs [hunk 1]
```diff
@@ -65,41 +66,48 @@ protected override void Append(LoggingEvent loggingEvent)
                 }
             }
 
-            var exception = loggingEvent.ExceptionObject ?? loggingEvent.MessageObject as Exception;
-            var evt = new SentryEvent(exception)
+            if (loggingEvent.Level > Breadcrumb)
```
Comment (bruno-garcia) [inline src/Sentry.Log4Net/SentryAppender.cs]: Perhaps we should have 2 event levels here like we do on NLog, Serilog, MEL.
The decision wouldn't have to be: Either breadcrumb or event.
We actually store the errors as breadcrumbs too so they get sent on the next event as part of the breadcrumb.

File: src/Sentry.Log4Net/SentryAppender.cs [hunk 2]
```diff
@@ -65,41 +66,48 @@ protected override void Append(LoggingEvent loggingEvent)
...
+                    {
+                        Name = Constants.SdkName,
+                        Version = NameAndVersion.Version
+                    },
+                    Logger = loggingEvent.LoggerName,
+                    Level = loggingEvent.ToSentryLevel()
+                };
 
-            evt.Sdk.AddPackage(ProtocolPackageName, NameAndVersion.Version);
+                evt.Sdk.AddPackage(ProtocolPackageName, NameAndVersion.Version);
```
Comment (bruno-garcia) [inline src/Sentry.Log4Net/SentryAppender.cs]: Are these tabs? :P

Comment (xt0rted) [reply src/Sentry.Log4Net/SentryAppender.cs]: According to [Render Whitespace on GitHub](https://github.com/glebm/render-whitespace-on-github) they're spaces.

image

Comment (bruno-garcia) [reply src/Sentry.Log4Net/SentryAppender.cs]: TIL :)

Comment (bruno-garcia) [reply src/Sentry.Log4Net/SentryAppender.cs]: I usually select it and spaces go one at a time (4), while tab would move at once.
I noticed it was 4 spaces but I'm not sure why it's showing as modified. I guess it was a tab before then :)

**Comment by bruno-garcia** [comment]
Feel free to reopen if you address the notes.

**Pull request closed without merging**